sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -614,7 +611,7 @@ public virtual async Task<string[]> GetPartitionIdsAsync(CancellationToken cance
         public virtual async ValueTask<EventDataBatch> CreateBatchAsync(CreateBatchOptions options,
                                                                         CancellationToken cancellationToken = default)
         {
-            options = options?.Clone() ?? DefaultCreateBatchOptions;
+            options = options?.Clone() ?? new CreateBatchOptions();
```

Comment:
Orthogonal, but I had forgotten that these option are modified when the maximum size allowed by the AMQP link is measured; because this instance is mutated, the shared member is not appropriate.

Reply:
Is there a test that validates this change?

Reply:
Yup, `CreateBatchDefaultsBatchOptions`.  It's been failing intermittently in the nightly runs since I introduced the `DefaultBatchOptions`.   It passes non-live runs in CI, and the race condition hid it during local live runs.